tests/ui_tests/app/page/common/configuration/page.py

```diff
@@ -348,6 +372,14 @@ def check_invalid_value_message(self, error_message: str):
     def is_save_btn_disabled(self):
         return self.find_element(self.locators.save_btn).get_attribute("disabled") == 'true'
 
+    @allure.step("Check save button status")
+    def check_save_btn_state(self, expected_state: bool):
+        assert (
+            not (self.is_save_btn_disabled()) == expected_state
+        ), f'Save button should{" not " if expected_state is False else " "}be disabled'
+        if expected_state:
+            self.save_config()
```

Comment:
This method isn't only checking status of save button

Reply:
renamed